test/unit/api/layer.test.js

```diff
@@ -4,14 +4,15 @@ import Layer from '../../../src/api/layer';
 
 describe('api/layer', () => {
     let source;
-    let style;
+    let style, style2;
```

Comment:
Why are you using `style2` instead of `style`?

Reply:
conceptually it makes more sense to blend to another style, when we improve the testing, we should probably need another style